### PR TITLE
Update prometheus message format

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Update prometheus message format

### CODE REVIEW

**Comment by huseyinbabal** [comment]
@mszostok tested locally, and it works 🚀 
Screen Shot 2023-02-03 at 10 02 33

File: internal/source/prometheus/source.go [hunk 1]
```diff
@@ -73,7 +74,14 @@ func (p *Source) consumeAlerts(ctx context.Context, config Config, ch chan<- []b
 			log.Errorf("failed to get alerts. %v", err)
 		}
 		for _, alert := range alerts {
-			msg := fmt.Sprintf("[%s][%s][%s] %s", PluginName, alert.Labels["alertname"], alert.State, alert.Annotations["description"])
+			msg := heredoc.Docf(`
+				Source: %s
+				Alert Name: %s
+				State: %s
+
+				Description:
```
Comment (pkosiec) [inline internal/source/prometheus/source.go:82]: What would happen if we used bold for keys? Like:

```suggestion
				*Source:* %s
				*Alert Name:* %s
				*State:* %s

				*Description:*
```

I suspect you already thought of this, and such formatting is not handled correctly in all comm platforms, correct?

Comment (huseyinbabal) [reply internal/source/prometheus/source.go:82]: @pkosiec it becomes bold 
Screen Shot 2023-02-03 at 10 49 22

Comment (pkosiec) [reply internal/source/prometheus/source.go:82]: On Slack and probably Discord + Mattermost yes, but not sure about MS Teams 🤔 I know @mszostok knows a few caveats about MS Teams, that's why I asked 😄 But hopefully we can apply the bold 🤞

**Comment by mszostok** [comment]
Thanks guys, I answered your comments, please take a look once again :)

@pkosiec @huseyinbabal

**Review by huseyinbabal — APPROVED** [review]

**Comment by mszostok** [comment]
Only MS Teams don't respect properly new lines:
Screenshot 2023-02-03 at 10 48 55
but it's still quite readable.

**Pull request merged**